src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+          {
+            "name": "action",
+            "in": "path",
+            "description": "The state transition to be executed",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "submit",
+              "approve",
+              "recall",
+              "sendback",
+              "cancel",
+              "close",
+              "reopen"
+            ]
+          },
+          {
+            "in": "body",
+            "name": "comment",
+            "description": "Only works with when the workflow action is ‘sendback’.",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/WorkflowAction"
+            }
+          },
+          {
+            "name": "companyId",
+            "in": "query",
+            "description": "The unique identifier of the company",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user performing the status transition",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceOfRequest"
+            }
+          },
+          "400": {
+            "description": "The request is invalid",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "401": {
+            "description": "Request is not authenticated"
+          },
+          "403": {
+            "description": "Principal is not allowed to perform operation"
+          },
+          "500": {
+            "description": "An unexpected error has prevented the operation",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/requests/{requestUuid}/expenses": {
+      "get": {
+        "tags": [
+          "Expense Resource"
+        ],
+        "summary": "Get expected expenses attached to a request",
+        "operationId": "listUsingGET",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "requestUuid",
+            "in": "path",
+            "description": "The unique identifier of the Request",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "description": "The unique identifier of the user viewing the expected expenses attached to a Request",
```

Comment:
The unique identifier of the user viewing the Expected Expenses attached to a Request